src/features/settings/pages/PracticeTeamPage.tsx

```
@@ -239,261 +235,215 @@ export const PracticeTeamPage = ({ className, onBack }: PracticeTeamPageProps) =
+            </SettingsCard>
+          )}
+        </SettingSection>
+
+        {isAdmin && (
+          <SettingSection
+            title="Invite a team member"
+            description="New members receive their own login and assistant thread. You control what they can see and do."
+          >
+            <SettingsCard className="max-w-[860px]">
+              <div className="font-serif text-lg mb-1">Send invitation</div>
+              <p className="text-xs text-dim mb-4">They'll receive an email with a link to join your workspace.</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)